src/components/Button.tsx

```diff
@@ -56,17 +56,27 @@ interface ButtonProps {
   outlined?: boolean
   small?: boolean
   icon?: boolean
+  url?: string
 }
 
 export default function ({
   outlined,
   children,
   small,
   icon,
+  url,
   ...rest
-}: React.HTMLAttributes<HTMLButtonElement> & ButtonProps) {
+}: Omit<React.HTMLAttributes<HTMLButtonElement>, 'icon'> & ButtonProps) {
   return (
-    <button className={button({ outlined, small, icon })} {...rest}>
+    <button
+      className={button({ outlined, small, icon })}
+      onClick={() => {
```

Comment:
Get rid of curly braces here

Reply:
I think it's better to leave them as they are. It's a standard syntax